css-basic2/css-basics_pokemon-page/css/styles.css

```diff
@@ -5,11 +5,14 @@
 /* Task 1: First of all we want to adjust how the size of our elements is calculated when working with CSS. Do you know how to switch from "content-box" to "border-box"? */
 
 /* your solution */
-
+* {
+  box-sizing: border-box;
+}
 /* Task 2: Let's work on some basic stylings that we want to apply to all elements of your code via inheritance. Which selector do we need for this? */
 
 /* add selector */
- {
+
+* {
   font-family: monospace;
   font-size: 1rem;
   line-height: 1.5;
```

Comment:
to have a cleaner code it's better to have only one class definition. so could you please have it all in one place?

* {
box-sizing: border-box;
font-family: monospace;
  font-size: 1rem;
  line-height: 1.5;
  }